extensions/amp-carousel/0.2/test/test-type-slides.js

```diff
@@ -165,16 +166,21 @@ describes.realWin(
 
     it('should style snap for container and content correctly', async () => {
       const carousel = await getCarousel({loop: true});
-      const slideWrappers = getSlideWrappers(carousel);
-      expect(slideWrappers.length).to.equal(5);
-
-      // Ensure that the content has the snap property not wrapper
-      // or else it will break scrolling animation.
-      for (let i = 0; i < slideWrappers.length; i++) {
-        expect(slideWrappers[i].style.scrollSnapAlign).to.equal('');
-        expect(slideWrappers[i].children[0].style.scrollSnapAlign).to.not.equal(
-          ''
-        );
+      const slides = carousel.querySelector('.i-amphtml-carousel-scroll')
+        .children;
+      expect(slides.length).to.equal(20);
+
+      // Ensure that the spacers have the snap property and not the
+      // slides.
+      for (let i = 0; i < slides.length; i++) {
+        const slide = slides[i];
+        if (slide.classList.contains('i-amphtml-carousel-spacer')) {
+          // type=slides is always center alignment.
+          expect(slide.style.scrollSnapAlign).to.equal(Alignment.CENTER);
```

Comment:
tiny nit: I'm unsure why we have the enum in the first place, but since this is a standard CSS keyword I'd say it's unnecessary to import the enum in this test:

```suggestion
          expect(slide.style.scrollSnapAlign).to.equal('center');
```